clients/fides-js/src/components/ConsentBannerButton.tsx

```diff
@@ -0,0 +1,20 @@
+import {h, FunctionComponent} from "preact";
+import {ButtonType} from "../lib/consent-types";
+
+interface ButtonProps {
+    buttonType: ButtonType;
+    label?: string,
+    onClick?: () => void
+}
+
+const ConsentBannerButton: FunctionComponent<ButtonProps> = ({ buttonType, label, onClick }) => (
+    <button
+        id={`fides-consent-banner-button-${buttonType.valueOf()}`}
+        className={`fides-consent-banner-button fides-consent-banner-button-${buttonType.valueOf()}`}
```

Comment:
at some point we will probably want to make constants out of these

Reply:
agree! Let's follow-up with these